ci/deploy_app.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+REGION="${REGION:=europe-west2}"
+
+CONCURRENCY="${CONCURRENCY:=80}"
+MIN_INSTANCES="${MIN_INSTANCES:=1}"
+MAX_INSTANCES="${MAX_INSTANCES:=1}"
+CPU="${CPU:=4}"
+MEMORY="${MEMORY:=4G}"
+
+WEB_SERVER_TYPE="${WEB_SERVER_TYPE:=gunicorn-threads}"
+WEB_SERVER_WORKERS="${WEB_SERVER_WORKERS:=7}"
+WEB_SERVER_THREADS="${WEB_SERVER_THREADS:=10}"
+WEB_SERVER_UWSGI_ASYNC_CORES="${WEB_SERVER_UWSGI_ASYNC_CORES:=10}"
+HTTP_KEEP_ALIVE="${HTTP_KEEP_ALIVE:=650}"
+
+EQ_KEYS_FILE="/keys/keys.yml"
+EQ_SECRETS_FILE="/secrets/secrets.yml"
+DATASTORE_USE_GRPC="${DATASTORE_USE_GRPC:=True}"
+EQ_STORAGE_BACKEND="${EQ_STORAGE_BACKEND:=datastore}"
+EQ_ENABLE_SECURE_SESSION_COOKIE="${EQ_ENABLE_SECURE_SESSION_COOKIE:=True}"
+EQ_RABBITMQ_ENABLED="${EQ_RABBITMQ_ENABLED:=False}"
+EQ_ENABLE_HTML_MINIFY="${EQ_ENABLE_HTML_MINIFY:=False}"
+EQ_RABBITMQ_HOST="${EQ_RABBITMQ_HOST:=rabbit}"
+EQ_RABBITMQ_HOST_SECONDARY="${EQ_RABBITMQ_HOST_SECONDARY:=rabbit}"
+EQ_QUESTIONNAIRE_STATE_TABLE_NAME="${EQ_QUESTIONNAIRE_STATE_TABLE_NAME:=questionnaire-state}"
+EQ_SESSION_TABLE_NAME="${EQ_SESSION_TABLE_NAME:=eq-session}"
+EQ_USED_JTI_CLAIM_TABLE_NAME="${EQ_USED_JTI_CLAIM_TABLE_NAME:=used-jti-claim}"
+EQ_SUBMISSION_BACKEND="${EQ_SUBMISSION_BACKEND:=gcs}"
+EQ_FEEDBACK_BACKEND="${EQ_FEEDBACK_BACKEND:=gcs}"
+EQ_PUBLISHER_BACKEND="${EQ_PUBLISHER_BACKEND:=pubsub}"
+EQ_SUBMISSION_CONFIRMATION_BACKEND="${EQ_SUBMISSION_CONFIRMATION_BACKEND:=cloud-tasks}"
+EQ_FULFILMENT_TOPIC_ID="${EQ_FULFILMENT_TOPIC_ID:=eq-fulfilment-topic}"
+EQ_INDIVIDUAL_RESPONSE_LIMIT="${EQ_INDIVIDUAL_RESPONSE_LIMIT:=75}"
+EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE="${EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE:=2021-04-28T02:00:00+00:00}"
+EQ_FEEDBACK_LIMIT="${EQ_FEEDBACK_LIMIT:=10}"
+EQ_GCS_SUBMISSION_BUCKET_ID="${PROJECT_ID}-submission"
+EQ_GCS_FEEDBACK_BUCKET_ID="${PROJECT_ID}-feedback"
```

Comment:
If values such as this will be hard coded to this format, then we should update the deploy yaml to not pass them and also the pipelines.yaml shouldn't pass it either as it wouldn't get used here.

Reply:
So these values are not passed in deploy.yaml and test pipeline.yaml, fixed👍